helm/kubernetes-kube-state-metrics-chart/templates/deployment.yaml

```diff
@@ -36,7 +36,7 @@ spec:
           initialDelaySeconds: 5
           timeoutSeconds: 5
       - name: addon-resizer
-        image: k8s.gcr.io/addon-resizer:1.8.3
+        image: quay.io/giantswarm/addon-resizer:1.8.3
```

Comment:
I think we use the templating here, cause in China we would use the retagged image on alyin, right @rossf7 ?

Reply:
Solved